content/docs/1.2.2/references/networking.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Longhorn Networking
```

Comment:
Thanks @jenting 
Agree. It is better using recommendation

Reply:
Yeah, like @innobead it seems all use TCP

Reply:
> Would use Longhorn Component Ports more clear?

I think Longhorn networking fit better since we also document the UNIX domain socket connections

> Do you think we need to add protocols as well?

Surely. Added. Thanks

> Probably the table format could reference https://kubernetes.io/docs/reference/ports-and-protocols/, it's more clear to read.

I think the current format is easier for the user to write networking policy. They just translate the ingress/egress from the doc to the networking policies

> Besides, what is Other Longhorn Manager means?
Yeah, like @innobead said. They are the manager pods on other nodes

Reply:
> They are the manager pods on other nodes

Ugh, okay. Thanks for the explanation. But still, a bit confused to me.